services/QuillLMS/engines/comprehension/spec/models/comprehension/automl_model_spec.rb

```diff
@@ -42,6 +42,14 @@ module Comprehension
         end
       end
 
+      context '#before_validation' do
+        it 'should strip whitespace from "automl_model_id"' do
+          automl_model = build(:comprehension_automl_model, automl_model_id: '  HAS_SPACES   ')
+          automl_model.valid?
+          expect(automl_model.automl_model_id).to eq('HAS_SPACES')
```

Comment:
Can we rename the dummy string literal? `'HAS_SPACES'` does not in fact have spaces.

Reply:
Fixed